pokeapp/src/components/FilterDropDown.js

```diff
@@ -41,6 +230,57 @@ const FilterDropDown = () => {
         {checkboxEl(ElementTypes.NORMAL, `Normal`)}
         {checkboxEl(ElementTypes.ROCK, `Rock`)}
       </div>
+
+        <div className="sort-container">
+
+          <div className="sort-type-container">
+            <div 
+              className="sort-btn"
+              onClick={
+                () => {
+                  setPokeDollarValue(false)
+                  setAToZ(true)
+                  setPokeID(false)
+                }
+              }
+            >
+                <h4>Alphabetical</h4>
+            </div>
+
+            <div 
+              className="sort-btn"
+              onClick={
+                () => {
+                  setPokeDollarValue(true)
+                  setAToZ(false)
+                  setPokeID(false)
+                }
+              }
+            >
+                <h4>Value</h4>
+            </div>
+
+            <div 
+              className="sort-btn"
+              onClick={
+                () => {
+                  setPokeDollarValue(false)
+                  setAToZ(false)
+                  setPokeID(true)
+                }
+              }
+            >
+                <h4>ID</h4>
+            </div>
+          </div>
+
+          {sortTypeOption()}
+          
+        </div>
```

Comment:
This looks like a great simplistic solution for handling the filter dropdown. Nice work!